examples/webextension/README.md

```diff
@@ -0,0 +1,29 @@
+This example webextension creates a toolbar button and sends a ping to GA each time it's clicked.
+
+## installation
+
+- Edit `background.js`, set the value of the `TRACKING_ID` const to your GA tracking ID
```

Comment:
RE: Mozilla ID vs. personal ID

- If we're making Firefox-y decisions based on this, maybe Mozilla ID
- Mozilla ID adds another dependency on the analytics team ([filing a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1253728))
- Personal ID has bus factor/SPOF risks
- Personal ID could also reduce overhead on graduation (e.g. don't have to transfer things, although [transferring ownership](http://webapps.stackexchange.com/questions/893/transfer-ownership-of-google-analytics-accounts) doesn't seem like a huge deal)

Maybe we suggest starting up with personal for quickness, then require a transfer to Mozilla before launch?